test/commands/app/build.test.js

```diff
@@ -123,14 +126,21 @@ const sampleAppConfig = {
   actions: { src: '/actions', dist: '/dist/actions', devRemote: false },
   root: path.resolve('test/__fixtures__/sample-app')
 }
+
+const createAppConfig = (aioConfig = {}, appFixtureName = 'legacy-app') => {
```

Comment:
I am a bit worried that most of the tests use the legacy-app mock test, which is a corner case, but we can modify this later if needed

Reply:
yeah, it was to make the existing unit-tests work without much modification. we can augment each test by running the command again with the exc-app-nui config